diff --git a/dist/types/jwks/remote.d.ts b/dist/types/jwks/remote.d.ts
index 7fd5fe1d858e94a9092d191237f30fd2b694671b..d4d9a3f3ddab5bda6b9045165854c952886205ae 100644
--- a/dist/types/jwks/remote.d.ts
+++ b/dist/types/jwks/remote.d.ts
@@ -1,4 +1,5 @@
 import type { KeyLike, JWSHeaderParameters, FlattenedJWSInput } from '../types';
+import { URL } from 'url';
 /** Options for the remote JSON Web Key Set. */
 export interface RemoteJWKSetOptions {
     /**